package.json

```diff
@@ -16,7 +16,9 @@
     "copy": "cp src/ui/index.html lib/ui/index.html",
     "lint": "eslint src/",
     "start": "gulp",
-    "test": "npm run lint"
+    "test": "npm run lint && mocha --compilers js:babel-register",
+    "test:watch": "mocha --compilers js:babel-register --watch",
+    "postinstall": "npm run build"
```

Comment:
We'll need to remove the `postinstall`.  I'm guessing this was added for your convenience while making changes, but including the build step here would necessitate moving `webpack` from `dependencies` to `devDependencies` (and introduce unnecessary complexity to `install`s).